### DISCUSSION TITLE
Tried platformio project import of Example

### DISCUSSION THREAD

**Discussion by rhenry74** [h5n1xp/Arduino_Sprite#1] [General]
Project Conversion fails with: Could not find the package with 'espressif/toolchain-riscv32-esp @ 8.4.0+2021r2-patch3' requirements for your system 'windows_amd64'

This is probably completely unrelated to this project. Just want you to know there is interest in getting this running via platformio.

Thanks

**Comment by h5n1xp** [comment]
Hey there! I don't have an RISCV chip at this time (though I do actually have a a RISCV emulator project called Ketura on the GitHub!), I would need to check if Expressif even have a RISCV toolchain, as far as I know all the ESP boards are ARM.

If you give me more details of the platform you are using I might be able to port Arduino_Sprite to it?
